diff --git a/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js b/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
index 63f7ef3..53177f9 100644
--- a/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
+++ b/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
@@ -52,7 +52,13 @@ function constructParcel(siteRoot) {
     targets: {
       root: {
         outputFormat: `commonjs`,
-        includeNodeModules: false,
+        includeNodeModules: [
+          'chalk',
+          'unist-util-find-after',
+          'unist-util-find-before',
+          'unist-util-map',
+          'unist-util-visit-parents',
+        ],
         sourceMap: false,
         engines: {
           node: `>= 14.15.0`
